SYCL/README.md

```diff
@@ -154,6 +157,9 @@ configure specific single test execution in the command line:
    host);
  * **sycl_be** - SYCL backend to be used (opencl, level_zero, cuda, hip);
  * **dump_ir** - if IR dumping is supported for compiler (True, False);
+ * **gpu_aot_target_opts** - defines additional options which are passed to AOT
+   compilation command line for Gpu device. It can be also set by CMake variable
+   GPU_AOT_TARGET_OPTS. If not specified "-device *" value is used.
```

Comment:
```suggestion
 * **gpu_aot_target_opts** - defines additional options which are passed to AOT
   compilation command line for GPU device. It can be also set by CMake variable
   GPU_AOT_TARGET_OPTS. If not specified "-device *" value is used.
```